src/protocol/logpoint.ts

```diff
@@ -251,11 +263,12 @@ export function setLogpointByURL(
   line: number,
   column: number,
   text: string,
-  condition: string
+  condition: string,
+  showInConsole: boolean = true
```

Comment:
i wonder if `setLogpoint` should take an object? these long lists of args gets sketchy

```
logGroupId, mapper, null, true
```